### PR TITLE
Adding README for running/configuring locally

### PR DESCRIPTION
**Pull request by nkatzman**
README about how to configure and run the `bin/server` locally

### CODE REVIEW

File: bin/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+To start the service, a few environment variables need to be set:
+
+### Required Environment Variables
+#### `screwdriver-plugin-login`
+| Environment Variable        |  Description           |
+| :------------- |:-------------|
+| $SECRET_JWT_PRIVATE_KEY| A private key uses for signing jwt tokens. Can be anything |
+| $SECRET_OAUTH_CLIENT_ID | The client id used for OAUTH with github. Look up [GitHub OAuth] for details |
+| $SECRET_OAUTH_CLIENT_SECRET | The client secret used for OAUTH with github |
+| $SECRET_PASSWORD | A password used for encrypting session, and oauth data. **Needs to be 32 characters long** |
```
Comment (FenrirUnbound) [inline bin/README.md]: > Needs to be 32 characters long

Exactly 32? Minimum 32?

**Comment by FenrirUnbound** [comment]
👍 

Looks good to me

swwrgwn - imgur

File: bin/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### Required Environment Variables
+#### `screwdriver-plugin-login`
+| Environment Variable        |  Description           |
+| :------------- |:-------------|
+| $SECRET_JWT_PRIVATE_KEY| A private key uses for signing jwt tokens. Can be anything |
+| $SECRET_OAUTH_CLIENT_ID | The client id used for OAUTH with github. Look up [GitHub OAuth] for details |
+| $SECRET_OAUTH_CLIENT_SECRET | The client secret used for OAUTH with github |
+| $SECRET_PASSWORD | A password used for encrypting session, and oauth data. **Needs to be minimum 32 characters** |
+| $IS_HTTPS | A flag to set if the server is running over https. Used as a flag for the oauth flow |
```
Comment (tkyi) [inline bin/README.md]: capitalize `oauth` for consistency

**Comment by stjohnjohnson** [comment]
👍 
mrw_my_brother_tells_me_that_my_invention_is_useless

**Pull request merged**